tests/dummy/app/routes/photo-blog.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+    model() {
```

Comment:
Please fix the indent spacing in this file to be consistent with the rest of the project.